python/ray/rllib/agents/impala/vtrace_policy_graph.py

```diff
@@ -235,15 +234,25 @@ def __init__(self,
             seq_lens=existing_seq_lens)
         unpacked_outputs = tf.split(
             self.model.outputs, self.output_hidden_shape, axis=1)
-        if self._is_discrete:
-            action_dist = dist_class(self.model.outputs)
-        else:
-            action_dist = dist_class(unpacked_outputs)
+        dist_inputs = self.model.outputs if self._is_discrete \
```

Comment:
I think that dist_inputs could just be unpacked_outputs? Not sure, think about it.

But whatever you do with model.outputs / unpacked outputs you should mirror it to behavior_logits / unpacked_behavior_logits - that is not the case currently.